_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
File added by mistake?

Reply:
Yeah I was toying around with Github pages, to see if we could generate the Documentation as github pages. That way we could include the docs in main repo. But it was a failed attempt. In the mean time my config file clutted the branch it seems.